qa/tasks/cephfs/test_admin.py

```diff
@@ -1610,10 +1610,11 @@ def test_single_path_authorize_on_nonalphanumeric_fsname(self):
         That fs authorize command works on filesystems with names having [_.-]
         characters
         """
-        self.skipTest('this test is broken ATM, see: '
-                      'https://tracker.ceph.com/issues/66077')
-
         self.mount_a.umount_wait(require_clean=True)
+        self.mount_b.umount_wait(require_clean=True)
+        # so that there's no confusion about whether or not we have
+        # self.mount_b
+        self.mount_b = None
```

Comment:
IMO you can just remove this. Later it will call the `mount_b.teardown`.

Reply:
Done.